sound/soc/sof/ipc4-topology.h

```diff
@@ -148,13 +148,15 @@ struct ipc4_pipeline_set_state_data {
  * struct sof_ipc4_available_audio_format - Available audio formats
  * @base_config: Available base config
  * @out_audio_fmt: Available output audio format
+ * @input_audio_fmts: Available input audio formats
```

Comment:
how do we know or track how many input_audio_fmts are used? This structure has int audio_fmt_num; but now it's unclear what the counter refers to: out_audio_fmt or in_audio_fmt?

edit: looks like the code assumes that audio_fmt_num refers to inputs only, maybe this should be documented line 154.